TicTacToe/BoardFactory.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace TicTacToe
+{
+    public class BoardFactory
+    {
+        public Board BuildBoard(int size)
+        {
+            Board newBoard = new Board(size);
+            return PopulateBoard(newBoard);
+        }
+
+        public Board PopulateBoard(Board board)
+        {
+            for (int space = 1; space <= board.Length(); space++)
+            {
+                board.gameBoard[space - 1] = space.ToString();
+            }
+            return board;
+        }
```

Comment:
I like this factory class since it has one responsibility and takes the onus off of other game components to set up the board.

One note: if you don't use PopulateBoard outside of this class, why not just make it a private method.

Reply:
Thanks, that's actually something I'll look at in each class, because I know there are other instances of that